src/NServiceBus.Core.Tests/Persistence/InMemory/SagaWithoutUniquePropertyData.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NServiceBus.Core.Tests.Persistence.InMemory
+{
+    using System;
+    using System.Threading.Tasks;
+
+    class SagaWithoutUniqueProperty : Saga<SagaWithoutUniquePropertyData>, IAmStartedByMessages<M13>
+    {
+        public Task Handle(M13 message, IMessageHandlerContext context)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<SagaWithoutUniquePropertyData> mapper)
+        {
+            //not implemented
+        }
+    }
+    public class SagaWithoutUniquePropertyData : ContainSagaData
+    {
+        public string NonUniqueString { get; set; }
+    }
+
+    class M13
```

Comment:
Helper class for the saga in the test. For some reason there are a bunch of them in the Core Persistence tests named things like M## so I just used the same naming pattern.